app/src/main/java/at/gammastrahlung/monopoly_app/network/MonopolyClient.java

```diff
@@ -61,11 +65,7 @@ public void joinGame(int gameId, String playerName) {
     public void getPlayers() {
         GameData gameData = GameData.getGameData();
 
-        webSocketClient.sendMessage(ClientMessage.builder()
-                .messagePath("players")
-                .player(gameData.getPlayer())
-                .message(null)
-                .build());
+        webSocketClient.sendMessage(ClientMessage.builder().messagePath("players").player(gameData.getPlayer()).message(null).build());
```

Comment:
Lombok builder is unreadable when written in a single line.
Revert this and all further ocurrences.